optimum/exporters/tasks.py

```diff
@@ -1717,9 +1856,9 @@ def infer_library_from_model(
         subfolder: str = "",
         revision: Optional[str] = None,
         cache_dir: str = HUGGINGFACE_HUB_CACHE,
-        library_name: Optional[str] = None,
         use_auth_token: Optional[Union[bool, str]] = None,
         token: Optional[Union[bool, str]] = None,
+        library_name: Optional[str] = None,
```

Comment:
Why?

Reply:
I just followed the pattern in the other `_infer_library_*` methods which is different than `_infer_task_*` for some reason. Another example is how the entrypoint `infer_task_from_model` can take model name/path/instance/class, but `infer_library_from_model` only takes name or path.
Would it make sense to standardize them ? the only hassle would be some changes on `optimum-subpackages` side.